embedchain/vectordb/elasticsearch.py

```diff
@@ -186,9 +196,11 @@ def query(
                 },
             }
         }
-        if "app_id" in where:
-            app_id = where["app_id"]
-            query["script_score"]["query"] = {"match": {"metadata.app_id": app_id}}
+
+        if len(where) > 0:

```

Comment:
Why checking based on the length? Why not do this:

```
if not where:
```

Reply:
You mean `if where:` will update it.